script/rspec_with_simplecov

```diff
@@ -24,15 +24,19 @@ begin
   # Simplecov emits some ruby warnings when loaded, so silence them.
   old_verbose, $VERBOSE = $VERBOSE, false
 
-  unless ENV['NO_COVERAGE'] || RUBY_VERSION.to_f < 2.1
+  skip_because_no_coverage = ENV['NO_COVERAGE']
+  skip_because_ruby_too_low = RUBY_VERSION.to_f < 2.1
+  skip_because_wrong_version = ENV['COVERAGE_ONLY_ON'] && ENV['COVERAGE_ONLY_ON'].to_f != RUBY_VERSION.to_f
```

Comment:
```suggestion
  skip_because_wrong_version = ENV['COVERAGE_ONLY_ON'].to_f != RUBY_VERSION.to_f
```
We always specify it, don’t we? Except for the legacy builds, but there a NO_COVERAGE is set.

Reply:
You can run the script locally - if you're not really intended to do so, I can switch this to checking and complaining that it's not set, but it's really the only way to get coverage information locally at the moment. So I wrote it intending that "if you specify the value and it doesn't match, skip simplecov"